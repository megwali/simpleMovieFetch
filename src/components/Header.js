import { Route, Switch } from 'react-router-dom';

const Header = () => (
  <Switch>
    <Route exact path="/">
      <h2>Home</h2>
    </Route>

    <Route path="/search">
      <h2>Search</h2>
    </Route>

    <Route path="/show">
      <h2>Show</h2>
    </Route>
  </Switch>
);

export default Header;
