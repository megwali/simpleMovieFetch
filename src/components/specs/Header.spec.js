import { create } from 'react-test-renderer';
import Header from '../Header';
jest.mock('react-router-dom', () => ({ Route: 'Route', Switch: 'Switch' }));

describe('Header', () => {
  it('renders', () => {
    const component = create(<Header />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
