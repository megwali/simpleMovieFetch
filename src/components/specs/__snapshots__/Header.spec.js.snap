// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<Switch>
  <Route
    exact={true}
    path="/"
  >
    <h2>
      Home
    </h2>
  </Route>
  <Route
    path="/search"
  >
    <h2>
      Search
    </h2>
  </Route>
  <Route
    path="/show"
  >
    <h2>
      Show
    </h2>
  </Route>
</Switch>
`;
